sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/implementation/models/RequestStatistics.java

```diff
@@ -5,16 +5,11 @@
 package com.azure.ai.textanalytics.implementation.models;
 
 import com.azure.core.annotation.Fluent;
-import com.fasterxml.jackson.annotation.JsonAnyGetter;
-import com.fasterxml.jackson.annotation.JsonAnySetter;
-import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
-import java.util.HashMap;
-import java.util.Map;
 
 /** if showStats=true was specified in the request this field will contain information about the request payload. */
 @Fluent
-public class RequestStatistics {
+public final class RequestStatistics {
```

Comment:
could this change be breaking in case some existing client code was extending it?

Reply:
[in the swagger, ](https://github.com/Azure/azure-sdk-for-java/pull/34779/files#diff-99ce08845da1127573360056ec9124a13b10c4d27de12b22c626348cb5f332aaL14)

The only child class is deleted so it is expected to generate this class to be a final class. The model is under '/implementation' folder and a generated model so no impact.